.github/scripts/pkg-test.sh

```diff
@@ -56,6 +56,18 @@ install_centos() {
   "$PKGMGR" install -y ${opts} curl nc jq || exit 1
```

Comment:
```suggestion
  "$PKGMGR" install -y "${opts}" curl nc jq || exit 1
```

Reply:
We actually need word splitting here so that the options listed in the variable get passed individually.